pom.xml

```diff
@@ -143,6 +143,14 @@
         <type>xml</type>
         <scope>runtime</scope>
       </dependency>
+      <dependency>
+        <groupId>pentaho</groupId>
+        <artifactId>pentaho-karaf-features</artifactId>
+        <classifier>standard</classifier>
```

Comment:
Wasn't this replaced with org.hitachivantara.karaf.features/pentaho-karaf-features-standard, already included above?

Reply:
Fixed